cryptographic_estimators/MQEstimator/series/hilbert.py

```diff
@@ -102,90 +135,106 @@ def precision(self):
             sage: H = HilbertSeries(5, [2]*7)
             sage: H.precision
             14
+
         """
-        return self.ring.default_prec()
+        x = self._gen
+        serie_repr = x.repr()
+        return int(serie_repr[serie_repr.rfind("=") + 1 : serie_repr.rfind(")")])
```

Comment:
this looks rather error prone. Wouldn't it be easier to store the `default_prec` from the constructor in a separate variable? And the return this variable here?

Reply:
Fully agree. I don't know why I forgot the 'self._prec' attribute.

Done!

Reply:
:+1: